packages/notifi-react-card/lib/components/subscription/subscription-card-views/EditCardView.tsx

```diff
@@ -45,86 +42,25 @@ export const EditCardView: React.FC<EditCardViewProps> = ({
   inputDisabled,
   inputSeparators,
   inputLabels,
+  hideAlertListPreview,
+  hideStartIcons,
 }) => {
   return (
     <div
       className={clsx('NotifiInputContainer', classNames?.NotifiInputContainer)}
     >
-      <AlertListPreview eventTypes={data.eventTypes} />
+      {hideAlertListPreview ? null : (
+        <AlertListPreview eventTypes={data.eventTypes} />
+      )}
```

Comment:
ditto -- i think we should try to achieve this with CSS overrides rather than props.